spec/bolt/transpiler_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'bolt/transpiler'
+
+describe Bolt::YamlTranspiler do
+  let(:basepath) { File.join(File.dirname(__FILE__), '..', 'fixtures', 'modules', 'yaml', 'plans') }
+  let(:plan) { described_class.transpile(plan_path) }
```

Comment:
It can result in confusing errors to declare a let without declaring it's dependencies.